### ISSUE TITLE
'hidden' does not seem to work as expected

### ISSUE THREAD

**Issue by xeroxoid** [ChrisWren/grunt-node-inspector#20]
Hi Chris, excellent node-inspector adaptation for grunt, thanks!

I seem to have problems using the 'hidden' option in the latest 0.1.5 version which supposedly supports it. I have tried many different values (from folders, to relative folders to specific files), but the files still appear in the UI and the code still breaks (even without breakpoints) inside those files.

Example grunt config:

```
'node-inspector': {
      custom: {
        options: {
          'web-host': 'localhost',
          'no-preload': true,
          'stack-trace-limit': 4,
          'hidden': ['node_modules', '../node_modules', '/node_modules/', '/node_modules', 'routes.js']
        }
      }
}
```

All other options seem to work fine though!

Thanks again for your time!

**Comment by fiznool** [comment]
I've added a PR #28 to fix this, as well as other fixes.

@ChrisWren if you are no longer using this plugin, would you be open to another maintainer? I'd be happy to take this on.

**Comment by ChrisWren** [comment]
@fiznool sorry for the late reply. Happy to add you as a maintainer? What is your npm user name?

**Comment by fiznool** [comment]
No problem. It's fiznool

**Comment by ChrisWren** [comment]
:smile: 
Thank you!
![1____repos_grunt-node-inspector__zsh_](https://cloud.githubusercontent.com/assets/2061937/9705766/5e8441da-5484-11e5-98d1-b78d8c39cc0e.png)

**Comment by mattpker** [comment]
@fiznool Did @ChrisWren give you write access to the repo to so that you can merge your pull request in. We are trying to update to Node v4 and this currently is the only package holding us back.

Thanks!

**Comment by fiznool** [comment]
Apologies I have yet to try. I'll see if I have access tomorrow (UK time)
to push a new release out.

**Comment by mattpker** [comment]
@fiznool Awesome, Thanks!

**Comment by ChrisWren** [comment]
If not, ping me. Thanks again for contributing!

**Comment by fiznool** [comment]
@ChrisWren I think it probably makes sense to give me write access to the repo, too, so I can keep the repo in sync with npm. What do you think?

I'm happy to take on ownership of the repo if you'd prefer to switch it over to me.

**Comment by simonkontschak** [comment]
Hi,
first of all thanks for implementing the node-inspector grunt task.
I also have the problem with the hidden option.

Each pattern i tried did not exclude folders/files.
So all libs (node/bower) are loaded and parsed which extremely slows down the startup.
Is there a fix in progress?

**Comment by fiznool** [comment]
@simonkontschak there is a fix #28 for this and other options. You could try using that branch directly to see if it solves your issue.

I'm working to try and get this integrated into the correct repo, the sticking point at the moment is the outstanding request for @ChrisWren to assign me write privileges to the repo so this can be fixed.

**Comment by simonkontschak** [comment]
Hi Tom,

thanks for your fast answer.
Your fix is working, good job.

Would be nice to have this fix in the correct repo asap.

**Comment by fiznool** [comment]
FYI I've rebased PR #28 to incorporate the recent changes on master.

**Comment by fiznool** [comment]
This should now be fixed with v0.4.0. Please re-open if you are still experiencing issues.
